test/ui/views/settings_view_test.dart

```diff
@@ -0,0 +1,49 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+// VIEW
+import 'package:notredame/ui/views/settings_view.dart';
+
+import '../../helpers.dart';
+
+void main() {
+  group('SettingsView - ', () {
+    setUp(() async {
+      setupNavigationServiceMock();
+      setupCacheManagerMock();
+      setupSettingsManagerMock();
+    });
+
+    tearDown(() {});
+
+    group('UI - ', () {
```

Comment:
Add some tests to check the opening of popups, and correct behaviour on tap for example the icon of the theme currently selected has multiple states (sunny, night, and a third one if I'm right), that should be tested :slightly_smiling_face: sorry, more tests to write

Reply:
done